src/components/Token.vue

```diff
@@ -1,16 +1,11 @@
 <template>
   <span class="d-inline-block v-align-middle line-height-0">
-    <img
-      v-if="!error"
-      :src="url"
-      :style="{
-        width: `${size || 22}px`,
-        height: `${size || 22}px`
-      }"
-      @error="error = true"
-      class="circle border line-height-0"
+    <UiAvatar
+      class="mr-2 mb-2"
+      :imgsrc="url"
+      :address="spaceAddress"
```

Comment:
What's the difference between seed and address? If there isn't difference we may want to use only one props, probably `seed` that should fit with all the case.

Reply:
If we add the address as a seed with will no longer match with the icon in Metamask and other wallets.

Reply:
The seed only has effect if the is no address provided.